src/Composer/ComposerWidget.vala

```diff
@@ -175,6 +178,14 @@ public class Mail.ComposerWidget : Gtk.Grid {
         web_view.query_command_state (command);
     }
 
+    private void on_remove_format () {
+        web_view.execute_editor_command ("removeformat");
+        web_view.execute_editor_command ("removeparaformat");
+        web_view.execute_editor_command ("unlink");
+        web_view.execute_editor_command ("backcolor", "#ffffff");
+        web_view.execute_editor_command ("forecolor", "#000000");
```

Comment:
Shouldn't removeformat already cover the backcolor and forecolor ?
Because that doesn't feel like "removing" things to set a color…